lib/Field/FieldRepeatableControls.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import { bool, number } from 'prop-types';
+import { bool, func, number } from 'prop-types';
 import cx from 'classnames';
 import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
 import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
 
 export function FieldRepeatableControls({
   repeatPosition,
   isLastRepeat,
+  deleteRepeatedField,
+  moveRepeatedFieldUp,
+  moveRepeatedFieldDown,
```

Comment:
Not a request for change here, as I think that this is a very opinionated comment and not anywhere near worth changing, but wanted to share for the heck of it anyway!

I like the convention of prefixing handlers with `on`, eg. `onDeleteRepeatedField`.

If we call it `deleteRepeatedField`, in my mental model we are suggesting that what is passed in must be the action to delete a field.

However, `onDeleteRepeatedField` to me is saying, this callback will be called when the UI button is pressed to delete a field. It doesn't matter what you pass in. I like this, as it suggests that this component is more flexible.